adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -212,16 +213,25 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
      * Perform authority validation.
      * True if the passed in authority is valid, false otherwise.
      */
-    private void validateAuthority(final URL authorityUrl, final UUID correlationId) throws AuthenticationException {
+    private void validateAuthority(final URL authorityUrl, @Nullable final String domain, final UUID correlationId) throws AuthenticationException {
```

Comment:
@weijjia You'll also see I introduced the `@Nullable` annotation to formalize that certain parameters are expected to potentially be null